doc/sphinx_source/using/tcl-commands.rst

```diff
@@ -3563,6 +3564,13 @@ The following is a list of bind types and how they work. Below each bind type is
 
   Module: irc
 
+(56) CHGHOST
+
+  bind chghost <flags> <mask> <proc>
+
+  procname <nick> <old user@host> <handle> <channel> <new user@host>
+
+  Description: triggered when a server sends an IRCv3 spec CHGHOST message to change a user's hostmask. The mask is matched against "#channel nick!user\@host" and can contain wildcards. The triggered proc will return the nick of the user who's hostmask changed; the hostmask the affected user had before the change, the handle of the affected user (or * if no handle is present), the channel the user was on when the bind triggered, and the new hostmask of the affected user. This bind will trigger once for each channel the user is on.
```

Comment:
"proc will return" -> proc will be passed?
It is unclear from this paragraph whether the mask is matched against the old nick!user@host or the new one

Reply:
Good catch- while the code current matches against the old user@host, I think this instead needs to match against the nw user@host, similar to how a NICK bind matches on the new nickname